src/cli/restore/restore.go

```diff
@@ -34,15 +34,41 @@ func AddCommands(cmd *cobra.Command) {
 
 const restoreCommand = "restore"
 
+//nolint:lll
+const restoreCommandExamples = `# Restore emails to their original location
+corso restore exchange \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--destination '/'
+
+# Restore OneDrive to a folder named "my_latest_restore"
+corso restore onedrive \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--mail-folder '*' \
+	--destination my_latest_restore
```

Comment:
Are we allowing non-root restores? Should this be `/my_latest_restore` ?

Reply:
destination always stores to the top level.  `my_latest` and `/my_latest` are treated as equivalent values.

Reply:
I would use the more explicit value in the example since it is more consistent with the in-place option